themes/salix/assets/css/alegreya.css

```diff
@@ -9,8 +9,8 @@
 @font-face {
 	font-family: 'Alegreya';
 	font-style: normal;
-	font-weight: 700;
+	font-weight: 900;
 	font-display: swap;
-	src: url(/alegreya/Alegreya-Bold.woff2) format('woff2');
+	src: url(/alegreya/Alegreya-Black.woff2) format('woff2');
```

Comment:
I think `ExtraBold` is 900, are we sure that this is the same weight? I can't get the only tool I know of for checking woff files working on this terrible system I'm stuck on, so I'm unsure.

Reply:
Hmmm I'll double check...

I just checked the previews on the google fonts site. black = 900, extra-bold = 800

Reply:
Opos, nevermind then; thanks!